modules/openapi-generator-maven-plugin/README.md

```diff
@@ -61,6 +61,7 @@ mvn clean compile
 - `generateSupportingFiles` - generate the supporting files (`true` by default)
 - `supportingFilesToGenerate` - A comma separated list of supporting files to generate.  All files is the default.
 - `skip` - skip code generation (`false` by default. Can also be set globally through the `codegen.skip` property)
+- `virtualService` - virtualized service code generation (`false` by default. Only available if `generator-name` is `spring`)
```

Comment:
I would suggest not documenting the option here as it's a generator-specified option but not a top-level option.

Reply:
I think it is important not prepare stuff in advance... Put it at spring config for the beginning. If you implement this in an other java-server, then we will see what makes sense.

Global level is wrong in my opinion: it will be very confusing for people generating a ruby or a node server.

Reply:
Make sense.. I had removed that from the top level.